test/test_dynamic_shape_models.py

```diff
@@ -0,0 +1,105 @@
+import sys
+import numpy as np
+import unittest
+import torch
+import torch_xla.core.xla_model as xm
+import torch_xla.debug.metrics as met
+
+xla_dev = xm.xla_device()
+
+
+class Feedforward(torch.nn.Module):
+
+  def __init__(self, input_size, hidden_size):
+    super().__init__()
+    self.input_size = input_size
+    self.hidden_size = hidden_size
+    self.fc1 = torch.nn.Linear(self.input_size, self.hidden_size)
+    self.fc1.weight.data.fill_(0.01)
+    self.fc1.bias.data.fill_(0.01)
+    self.relu = torch.nn.ReLU()
+    self.fc2 = torch.nn.Linear(self.hidden_size, 1)
+    self.fc2.weight.data.fill_(0.01)
+    self.fc2.bias.data.fill_(0.01)
+    self.sigmoid = torch.nn.Sigmoid()
+
+  def forward(self, x):
+    hidden = self.fc1(x)
+    relu = self.relu(hidden)
+    output = self.fc2(relu)
+    output = self.sigmoid(output)
+    return output
+
+
+class TestDynamicShapeModels(unittest.TestCase):
+
+  def test_forward_pass_nn_model_correctness(self):
```

Comment:
maybe `test_forward_pass_dynamic_input_correctness` ? The model itself is not dynamic, but the input are.

Reply:
done.